xfields/ibs/_equilibrium_emittance.py

```diff
@@ -127,12 +131,12 @@ def _ibs_rates_and_emittance_derivatives(
     # Ask to compute the IBS growth rates (this function logs so no need to do it here)
     ibs_growth_rates = twiss.get_ibs_growth_rates(
         formalism=formalism,
-        total_beam_intensity=bunch_intensity,
+        total_beam_intensity=total_beam_intensity,
         gemitt_x=input_emittance_x,
         gemitt_y=input_emittance_y,
         sigma_delta=sigma_delta,
         bunch_length=sigma_zeta,  # 1 sigma_{zeta,RMS} bunch length
-        bunched=True,
+        **kwargs,
```

Comment:
I removed `bunched=True` since it is default in IBS rates calculation. Since we pass `kwargs` from the top of the calls to here, as user can call the public function and provide `bunched=False` which will be sent to IBS rates just fine.